schema/synthetix/view_trades.sql

```diff
@@ -1,55 +1,35 @@
-CREATE OR REPLACE VIEW synthetix.view_trades (
-    block_time,
-    project,
-    version,
-    trader_a,
-    trader_b,
-    token_a_amount_raw,
-    token_b_amount_raw,
-    token_a_address,
-    token_b_address,
-    exchange_contract_address,
-    tx_hash,
-    trace_address,
-    evt_index,
-    token_a_amount,
-    token_b_amount,
-    token_a_usd_price,
-    token_b_usd_price
-) AS SELECT
+BEGIN;
+DROP MATERIALIZED VIEW IF EXISTS synthetix.view_trades;
+CREATE MATERIALIZED VIEW synthetix.view_trades AS
+WITH 
+rates AS (
+    SELECT currency_key, currency_rate, evt_block_time
+    FROM synthetix."ExchangeRates_evt_RatesUpdated" r, unnest("currencyKeys", "newRates") as u(currency_key, currency_rate)
+)
```

Comment:
Sorry if I was unclear previously, but can we just make this part `synthetix.view_rates`?